templates/trip/history.html.twig

```diff
@@ -0,0 +1,17 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}History{% endblock %}
+
+{% block body %}
+
+<h1>Historique</h1>
+
+{% for histories in history %}
```

Comment:
I guess you want to write:

    {% for trip in history %}

it looks strange to have one history becoming several history